src/routes/embed/+layout.ts

```diff
@@ -1 +1 @@
-export const ssr = true;
```

Comment:
Are you intentionally disabling SSR for embed routes?

Reply:
No, SSR is the default, so this was unneeded.